npm/metrics/test-util.go

```diff
@@ -0,0 +1,22 @@
+package metrics
```

Comment:
Can you rename this file as util_test.go (and change format for all test files as <name>_test.go)?